modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -31,6 +31,20 @@ Yes, Object Store v2 is in the same region as the worker where the application i
 
 IMPORTANT: If after the first deploy, you move the app to a different region, Object Store v2 remains in the original region to avoid data loss. Your use of Object Store v2 never moves from one region to another. When you need to deploy both an app and object store to a different region, delete the application and re-upload the app into the new region. This action loses all the data that may exist in the storage.
 
+== How are Object Store entries divided among workers?
+
+All entries that you put in `_defaultUserObjectStore` are available to all workers for the same app. For example, if you had 50 entries and 5 workers, all 50 entries would be available to all the workers in the app and the Object Store v2 service handles storing all the data without any impact on the workers.
```

Comment:
I would add something here reminding people that OSv2 is a remote service that is separate from the workers.

Reply:
See comment below for changed wording